lib/Doctrine/Common/Cache/FileCache.php

```diff
@@ -126,9 +113,9 @@ protected function getFilename($id)
     {
         return $this->directory
             . DIRECTORY_SEPARATOR
-            . implode(str_split(hash('sha256', $id), 2), DIRECTORY_SEPARATOR)
+            . substr(hash('sha256', $id), 0, 2)
```

Comment:
This is most probably enough indeed.
@guilhermeblanco @zeroedin-bill what is your opinion?